lib/travis/api/v3/queries/enterprise_license.rb

```diff
@@ -0,0 +1,13 @@
+module Travis::API::V3
+  class Queries::EnterpriseLicense < Query
+    def active_users(expiration_time)
+      this_year = Date.parse(expiration_time)
+      last_year = this_year - 365
+
+      Models::Email
+      .joins("INNER JOIN commits c ON LOWER(c.committer_email) = LOWER(emails.email)")
```

Comment:
I added the following indexes, which the QUERY PLAN said it was using:
`CREATE INDEX CONCURRENTLY index_commits_on_created_at ON commits (created_at);`
`CREATE INDEX CONCURRENTLY index_commits_on_lower_committer_email ON commits (lower(committer_email));`

These improved the query time, but it still needs to scan sequentially over millions of commits, so is very time consuming (10 mins) and therefore not feasible to trigger form this endpoint. We are considering a different implementation in order to resolve this...